Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -271,7 +273,10 @@ open class MessageContentCell: MessageCollectionViewCell {
     /// Positions the cell's accessory view.
     /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
     open func layoutAccessoryView(with attributes: MessagesCollectionViewLayoutAttributes) {
-        var y = (bounds.height - attributes.accessoryViewSize.height) / 2
+        
+        // Accessory view aligned to the middle of the messageContainerView
+        var y = (messageContainerView.bounds.height - attributes.accessoryViewSize.height) / 2
+        y += messageContainerView.frame.origin.y
```

Comment:
Pretty sure the `cellContentSize` method will be incorrect in terms of height due to this new alignment

Reply:
Maybe not actually 🤔 Need to review in more depth

Reply:
@nathantannar4 I think your logic for this is:
```swift
var y = messageContainerView.frame.midY - (attributes.avatarSize.height/2)
```
This is a tough area of the codebase I know

Reply:
@SD10 Made this change